WordPressAuthenticator/Unified Auth/View Related/Reusable Views/TextFieldTableViewCell.swift

```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+
+/// TextFieldTableViewCell: a textfield with a custom border line in a UITableViewCell.
+///
+class TextFieldTableViewCell: UITableViewCell {
+
+    public static let reuseIdentifier = "TextFieldTableViewCell"
+
+    @IBOutlet var borderView: UIView!
+    @IBOutlet var textField: UITextField!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        borderView.backgroundColor = WordPressAuthenticator.shared.unifiedStyle?.borderColor
```

Comment:
What happens if `unifiedStyle` is undefined (since it optional)? Maybe it should fallback to `WordPressAuthenticator.shared.style.primaryNormalBorderColor`?